File: easy/01 - Two-Number-Sum/01.py
def two_number_sum(array, targetSum):
    out = []
    for i in range(len(array)):
        num = array[i]
        difference = targetSum - num
        if difference in array[i + 1 :]:
            out = out + [difference, num]
    return out


print("solution:", two_number_sum([3, 5, -4, 8, 11, 1, -1, 6], targetSum=10))

File: easy/01 - Two-Number-Sum/01.js
function twoNumberSum(array, targetSum) {
    const out = []
    for (i=0; i<array.length; i++) {
      const num = array[i]
      const difference = targetSum - num;
      if (array.slice(i+1).includes(difference)) {
        out.push(difference, num)
      }
    }
    return out
  }

console.log('solution:', twoNumberSum([3, 5, -4, 8, 11, 1, -1, 6] , targetSum = 10))